src/EditorFeatures/Test2/GoToDefinition/VisualBasicGoToDefinitionTests.vb

```diff
@@ -564,7 +564,7 @@ End Class
 
 Module Ex
     <System.Runtime.CompilerServices.Extension()>
-    Public Sub TestExt(Of T)(ex As T)
+    public async function TestExt(Of T)(ex As T)
```

Comment:
This is editing the test contents here, and may actually break that test?

Reply:
damn!  i tried to be so careful to not do that.